src/sentry/api/endpoints/team_alerts_triggered.py

```diff
@@ -0,0 +1,51 @@
+from django.db.models import Count, Q
+from django.db.models.functions import TruncDay
+from rest_framework.response import Response
+
+from sentry.api.base import EnvironmentMixin
+from sentry.api.bases.team import TeamEndpoint
+from sentry.api.utils import get_date_range_from_params
+from sentry.incidents.models import (
+    IncidentActivity,
+    IncidentActivityType,
+    IncidentProject,
+    IncidentStatus,
+)
+from sentry.models import Project
+
+
+class TeamAlertsTriggeredEndpoint(TeamEndpoint, EnvironmentMixin):
+    def get(self, request, team):
+        """
+        Return a time-bucketed (by day) count of triggered alerts owned by a given team.
+        """
+        project_list = Project.objects.get_for_team_ids([team.id])
+        owner_ids = [team.actor_id] + [om.user.actor_id for om in team.member_set]
```

Comment:
`[om.user.actor_id for om in team.member_set]` -  I think this will make a query for each `.user` call. You should be able to do `team.member_set.select_related("user")` to prefetch this.

Actually, I wonder if `team.member_set.values_list("user__actor_id", flat=True)` would work even better, since then we wouldn't need to bring the models into memory. This isn't that important to do, just an extra thought.

Reply:
Made the change